### DISCUSSION TITLE
New type of panel in dashboards (Insights)

### DISCUSSION THREAD

**Discussion by lodgmar** [directus/directus#23403] [Feature Requests]
### Summary

It would be great to have a new kind of panel, mix of "Metrics" and "Global variable". That is, you can define a Metric with an aggregate function (sum, count or whatever) and use it as a variable for other panels (like Meter Chart's maximum value). This panel could be hidden to users, if needed.

### Motivation

Meter Chart is almost useless because it needs a fixed maximum value. I was thinking how it could have a variable maximum value, and that's how I got here. But a "Metric variable" would be of great help in many other situations, mainly for maximum values of graphs' axes.

### Detailed Design

Apologies, I don't know Directus' internals, so I can't help with designs.

### Must Have:
- Aggregations functions available: `Count`, `Sum`, `Max`, `Min`

### Should Have:
- ...

### Could Have:
- ...

### Won't Have:
- ...

### Drawbacks

I don't know how Directus works internally, but I guess this is an easy functionality to implement..., so no drawbacks 😃

### Alternatives

...

### Adoption Strategy

No need for migrations. Just a new funcionality.

**Comment by LewisMcReu** [comment]
Alternatively allowing the use of the output of Metrics panels as variables in other panels would do the exact same. 👍

**Reply by lodgmar** [reply]
Yes indeed 👍
